Automattic/newspack-plugin
# Add migration option to onboarding

### All Submissions:






### Changes proposed in this Pull Request:

* update `@wordpress/icons` npm package to latest version (5.0.0)
* add `ButtonCard` and new `TextControl` to Welcome Screen
* add new `isPressed` prop to `ButtonCard`
* Add migration option to onboarding which imports WordPress sites.

### How to test the changes in this Pull Request:

1. Start with a clean Newspack site (visit `wp-admin/?newspack_reset=starter-content` and `wp-admin/?newspack_reset=reset`)
2. Run the onboarding wizard at `wp-admin/admin.php?page=newspack-setup-wizard#/`. Observe two options now:

![screenshot-welcome-screen](https://user-images.githubusercontent.com/177929/128049830-b61b006a-93f8-4d07-80a3-f68ac8d61b24.png)

3.  Select the migrate option. Put in the URL **of a WordPress site**:

<img width="452" alt="Screen Shot 2021-09-16 at 11 13 16 AM" src="https://user-images.githubusercontent.com/7317227/133670327-26f76a67-a1af-49eb-86c3-2d0c65dcf68a.png">

4. Run the migration. Once the posts have finished importing, visit the site. You should see a homepage with all imported content. (Optional: the homepage layouts on the Design tab should still work nicely)

<img width="612" alt="Screen Shot 2021-09-16 at 11 15 52 AM" src="https://user-images.githubusercontent.com/7317227/133670656-de60773e-81c1-4da1-a2af-577929a9fa4d.png">

5. Visit `wp-admin/?newspack_reset=starter-content`. Observe starter content deletes nicely.
6. Visit `wp-admin/?newspack_reset=reset`. Run the wizard again, but with the Start a New Site and Demo Content option to make sure my refactoring didn't break anything.

### Other information:
